docs/code-of-conduct.rst

```diff
@@ -29,6 +26,45 @@ This code of conduct applies to all spaces associated with the Write the Docs co
   * Advocating for, or encouraging, any of the above behavior.
   * Repeated harassment of others. In general, if someone asks you to stop, then stop.
 
-* **When we disagree, try to understand why.** Disagreements, both social and technical, happen all the time and Write the Docs is no exception. It is important that we resolve disagreements and differing views constructively. Remember that we’re different. The strength of Write the Docs comes from its varied community -- people from a wide range of backgrounds. Different people have different perspectives on issues. Being unable to understand why someone holds a viewpoint doesn’t mean that they’re wrong. And don’t forget that it is human to err and blaming each other doesn’t get us anywhere. Instead, focus on helping to resolve issues and learning from mistakes.
+Where does the code of conduct apply
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This code of conduct applies to all spaces managed by Write the Docs. This includes:
+
+* Conferences (including social events and peripheral activities)
+* Unconferences and sprints
+* Meetups
+* Workshops
+* Presentation materials used in talks or sessions
+* Slack
+* Mailing lists
+* GitHub
+* Twitter hashtag
```

Comment:
This is a hard one, but we can at least enforce bad actors on twitter by banning them from the event.